docs/data/material/guides/interoperability/interoperability.md

```diff
@@ -682,9 +684,15 @@ export default function PlainCssPriority() {
 
 **Note:** If you are using styled-components and have `StyleSheetManager` with a custom `target`, make sure that the target is the first element in the HTML `<head>`. If you are curious to see how it can be done, you can take a look at the [`StyledEngineProvider`](https://github.com/mui/material-ui/blob/master/packages/mui-styled-engine-sc/src/StyledEngineProvider/StyledEngineProvider.js) implementation in the `@mui/styled-engine-sc` package.
 
+### Usage
+
 Now it's all set up and you can start using Tailwind CSS on the MUI components!
 
-**App.js**
+{{"demo": "StyledComponents.js", "hideToolbar": true}}
+
+[![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/github-ndkshy?file=pages%2Findex.tsx)
```

Comment:
Should we have this in the `examples` too? What was the reason you decided this to be available on StackBlitz only?

Reply:
I didn't know we had this folder.

Reply:
> Should we have this in the examples too?

The demo doesn't work with them, for some reason

> What was the reason you decided this to be available on StackBlitz only?

Codesandbox is slow as hell with Next.js